packages/animations/test/page_transition_switcher_test.dart

```diff
@@ -0,0 +1,622 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:animations/animations.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('transitions in a new child.', (WidgetTester tester) async {
```

Comment:
Should verify that transitioning to/from a null child works too

Reply:
I believe there is a subsequent test for this called "handles null children"